src/main/webapp/i18n/en/exercise-actions.json

```diff
@@ -10,12 +10,20 @@
             "statistics": "Quiz Statistics",
             "startExercise": "Start exercise",
             "resumeExercise": "Resume exercise",
+            "resumeExercisePractice": "Resume practice in exercise",
             "resumeExerciseError": "Error trying to resume the exercise",
             "openCodeEditor": "Open code editor",
+            "openPracticeCodeEditor": "Open practice code editor",
+            "openGradedCodeEditor": "Open graded code editor",
+            "practiceMode": "Practice Mode",
+            "choseGradedMode": "Graded participation chosen",
             "openModelingEditor": "Open modeling editor",
             "importIntoIntelliJ": "Open in IntelliJ",
+            "practiceModeTooltip": "The practice mode allows you to keep working on the exercise after the submission due date and keep practicing. This will not affect your grading! It will setup a separate repository for you that you need to clone separately.",
```

Comment:
```suggestion
            "practiceModeTooltip": "The practice mode allows you to keep working on the exercise after the submission due date for self-practice. This will not affect your grading! Artemis will setup a separate repository for you to clone.",
```